src/main/java/com/esri/samples/localserver/local_server_services/LocalServerServicesController.java

```diff
@@ -123,6 +135,32 @@ private void handleStartSelectedService() {
       }
     }
   }
+  
+  @FXML
+  private void handleSelectServiceOption() {
+    String selected = serviceOptions.getSelectionModel().getSelectedItem();
+    String pathStart = "./samples-data/local_server/";
+    switch (selected) {
+      case "Map Service":
+      case "Feature Service":
+        lblSelectedPackage.setText(pathStart + "PointsofInterest.mpk");
+        break;
+      case "Geoprocessing Service":
+        lblSelectedPackage.setText(pathStart +  "MessageInABottle.gpk");
+        break;
+      default:
+        break;
+    }
+  }
+  
+  @FXML
+  private void handleSelectPackage() {
+    File selectedFile = LocalServerServicesSample.openFileDialog();

```

Comment:
You can get the stage from the scene like so: 
```java
File selectedFile = new FileChooser().showOpenDialog(btnStartServer.getScene().getWindow());
```
No need for the static method and field on the main class.